src/storage-client/src/controller.rs

```diff
@@ -183,9 +185,8 @@ impl<T> CollectionDescription<T> {
                 }
                 result.push(ingestion.remap_collection_id);
             }
-            DataSource::Introspection(_) | DataSource::Progress => {
-                // Introspection, Progress sources have no dependencies, for
-                // now.
+            DataSource::Webhook | DataSource::Introspection(_) | DataSource::Progress => {
+                // Introspection, Progress, and Webhook sources have no dependencies, for now.
```

Comment:
Once we have validation in webhooks, we'll need to determine if it has any non-system dependencies, e.g. someone used one of their own custom types in the body. Nothing to do now but something to be aware of.

Reply:
Good callout! I added a TODO here